config/esbuild/build.js

```diff
@@ -125,7 +125,8 @@ const config = {
     },
   ],
   //loader: { ".js": "jsx" },
-  target: ["es6", "chrome58", "firefox57", "safari11", "edge18"],
+  // target: ["es6", "chrome58", "firefox57", "safari11", "edge18"],
+  target: ["es2020"],
```

Comment:
why did you remove all other ones? did you align it with @andypf ?

Reply:
Please no changes outside of your plugin without discussing it first!

Reply:
reverted back. not needed for this branch.